### PR TITLE
added support for timestamp types

### PR DESCRIPTION
**Pull request by MaksymSkrynnikov**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

By default REALTIME_LAG calculated as NOW (epoch millis) - MAX(timestamp) from SQL, added support to be able to operate not only in milliseconds but in hours as well.

### CODE REVIEW

**Review by rossoha — APPROVED** [review]

File: bandarlog/src/main/scala/com/aol/one/dwh/bandarlog/providers/SqlProvider.scala [hunk 1]
```diff
@@ -49,10 +51,17 @@ class GlueTimestampProvider(connector: GlueConnector, table: Table) extends Time
...
   * Provides current time in milliseconds
   */
-class CurrentTimestampProvider extends TimestampProvider {
+class CurrentTimestampProvider(timestampType: Option[String]) extends TimestampProvider {
 
   override def provide(): Value[Timestamp] = {
-    AtomicValue(Option(System.currentTimeMillis()))
+    def now(): Long = {
+      timestampType match {
+        case Some("hour") => Instant.now().getEpochSecond / 3600L
```
Comment (gemelen) [inline bandarlog/src/main/scala/com/aol/one/dwh/bandarlog/providers/SqlProvider.scala:59]: I'm sure that input could be treated more liberal than exact match of "hour" string.

Comment (gemelen) [reply bandarlog/src/main/scala/com/aol/one/dwh/bandarlog/providers/SqlProvider.scala:59]: Also, what does this formula mean?
I do not understand how timestamp divided by constant is linked to hour range, so could you please explain it (probably with comment in source)?

File: infra/src/main/scala/com/aol/one/dwh/infra/config/Configs.scala [hunk 2]
```diff
@@ -103,7 +103,13 @@ case class Topic(id: String, values: Set[String], groupId: String)
   * @param formats - format of columns (e.g., yyyy, MM, dd, HH:mm:ss) for datetime column
   * @param tag - an alternative tag to use for metrics instead of default, which is a table name
   */
-case class Table(table: String, columns: List[String], filters: Option[List[Filter]], formats: Option[List[String]], tag: Option[String])
+case class Table(table: String,
```
Comment (gemelen) [inline infra/src/main/scala/com/aol/one/dwh/infra/config/Configs.scala:106]: It seems to be an empty line was added here.

File: infra/src/main/scala/com/aol/one/dwh/infra/config/RichConfig.scala [hunk 3]
```diff
@@ -209,10 +209,10 @@ object RichConfig {
     def getTables: Seq[(Table, Table)] = {
       underlying.getObjectList("tables").map { obj =>
 
-      val columnType = obj.toConfig.getOptionalString("column-type").getOrElse("")
+      val columnType = obj.toConfig.getOptionalString("column-type").map(_.split(":")).getOrElse(Array("", ""))
```
Comment (gemelen) [inline infra/src/main/scala/com/aol/one/dwh/infra/config/RichConfig.scala:212]: Please, move chained operations in new lines.

**Review by gemelen — CHANGES_REQUESTED** [review]

File: bandarlog/src/main/scala/com/aol/one/dwh/bandarlog/providers/SqlProvider.scala (hunk 1 above)
Comment (MaksymSkrynnikov) [reply bandarlog/src/main/scala/com/aol/one/dwh/bandarlog/providers/SqlProvider.scala]: Updated

**Review by gemelen — APPROVED** [review]

**Pull request merged**